foqus_lib/gui/ouu/ouuSetupFrame.py

```diff
@@ -680,7 +680,7 @@ def clearPlots(self):
         self.objXPoints = []
         self.objYPoints = []
         if "objFigAx" in self.__dict__ and len(self.objFigAx.lines) > 0:
-            self.objFigAx.lines = []
+            self.objFigAx.lines.clear()
```

Comment:
Seems reasonable, curious if this is related to the rest of this PR or just something you spotted.

Reply:
Good question, yeah, this is something that I believe is completely unrelated, but it did cause an error (`Can't set attribute`) if left as it was. My best guess is that at some point the `lines` attribute of that object (whatever is returned by `matplotlib.Figure.make_subplots()`) was turned into a read-only [property](https://docs.python.org/3/library/functions.html#property).